commit 9899b6b1d6d0c6ec99ef3c33a692aee217d3976b (refs/changes/33/5333/2)
Author: Patrick Mooney <pmooney@pfmooney.com>
Date:   2019-01-09T14:46:28+00:00 (9 months ago)
    
    OS-7493 Welcome to 2019
    Reviewed by: Kody Kantor <kody.kantor@joyent.com>
    Approved by: Kody Kantor <kody.kantor@joyent.com>

diff --git a/usr/src/uts/common/os/logsubr.c b/usr/src/uts/common/os/logsubr.c
index 946f9d50ae..06c03dd38e 100644
--- a/usr/src/uts/common/os/logsubr.c
+++ b/usr/src/uts/common/os/logsubr.c
@@ -22,7 +22,7 @@
 /*
  * Copyright (c) 2013 Gary Mills
  * Copyright (c) 1998, 2010, Oracle and/or its affiliates. All rights reserved.
- * Copyright 2018 Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 #include <sys/types.h>
@@ -250,7 +250,7 @@ log_init(void)
 	 */
 	printf("\rSunOS Release %s Version %s %u-bit\n",
 	    utsname.release, utsname.version, NBBY * (uint_t)sizeof (void *));
-	printf("Copyright (c) 2010-2018, Joyent Inc. All rights reserved.\n");
+	printf("Copyright (c) 2010-2019, Joyent Inc. All rights reserved.\n");
 #ifdef DEBUG
 	printf("DEBUG enabled\n");
 #endif
